### PR TITLE
[Improve] yarnQueue check exists improvement

### PR DESCRIPTION
**Pull request by wolfboys**
[Improve] yarnQueue check exists improvement

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @RocMarshal PTAL, thx.

**Comment by RocMarshal** [comment]
Thanks for the improvement👍. please let me have a look~

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/YarnQueueServiceImpl.java [hunk 1]
```diff
@@ -132,16 +126,21 @@ public boolean createYarnQueue(YarnQueue yarnQueue) {
...
     YarnQueue queueFromDB = getYarnQueueByIdWithPreconditions(yarnQueue);
+
+    // 1) no data to update
     if (StringUtils.equals(yarnQueue.getQueueLabel(), queueFromDB.getQueueLabel())
         && StringUtils.equals(yarnQueue.getDescription(), queueFromDB.getDescription())) {
-      throw new ApiAlertException(NO_NEED_UPDATE_HINT);
+      return;
     }
+
+    // 2 update description
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/YarnQueueServiceImpl.java:136]: nice comment ~

File: streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/YarnQueueServiceTest.java
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/YarnQueueServiceTest.java:1]: From the test results, it maybe stuck in these lines(122-124) of code.
Apart from this issue, LGTM.

Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/YarnQueueServiceTest.java:1]: From the test results, it maybe stuck in these lines of code.
Apart from this issue, LGTM.

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/YarnQueueServiceImpl.java [hunk 2]
```diff
@@ -132,16 +126,21 @@ public boolean createYarnQueue(YarnQueue yarnQueue) {
   public void updateYarnQueue(YarnQueue yarnQueue) {
 
     YarnQueue queueFromDB = getYarnQueueByIdWithPreconditions(yarnQueue);
+
+    // 1) no data to update
     if (StringUtils.equals(yarnQueue.getQueueLabel(), queueFromDB.getQueueLabel())
         && StringUtils.equals(yarnQueue.getDescription(), queueFromDB.getDescription())) {
-      throw new ApiAlertException(NO_NEED_UPDATE_HINT);
+      return;
```
Comment (RocMarshal) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/YarnQueueServiceImpl.java:133]: I'd like it.👍

**Review by RocMarshal — COMMENTED** [review]
@wolfboys Thanks for driving the discussion for the MR review. 
I left a few comments. PTAL.
Thanks so much for the improvement~

File: streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/YarnQueueServiceTest.java
Comment (wolfboys) [reply streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/YarnQueueServiceTest.java:1]: > From the test results, it maybe stuck in these lines(122-124) of code. Apart from this issue, LGTM.

updated.

**Comment by RocMarshal** [comment]
@wolfboys Thanks for the update. LGTM +1. (non-binding.)

**Review by RocMarshal — APPROVED** [review]

**Review by lvshaokang — APPROVED** [review]
LGTM

**Comment by lvshaokang** [comment]
@RocMarshal Merged.

**Pull request merged**